index.html

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+
```

Comment:
### Your Summary
![summary](https://user-images.githubusercontent.com/93039359/185357005-b2855b4e-a157-4225-866f-9bac3b619770.png)


### Descriptive Summary Example :heavy_check_mark:
![hammas-summary](https://user-images.githubusercontent.com/93039359/176921727-17499afe-c29c-47f8-823e-a3cfc70feae1.png)

**[Optional]**
- [ ] You don't have a descriptive summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking: